### ISSUE TITLE
Responding very slow when resizing window

### ISSUE THREAD

**Issue by bhalial** [adamwulf/Columnizer-jQuery-Plugin#177]
Hi, as the title says, the responsiveness of the script is really slow when trying to test the columns.

Otherwise it works like a dream, it's amazing! But I can't get away with such a slow response time on the calculating of the columns.

Any ideas? I really wish to use this in my projects.

**Comment by sidaurukfreddy** [comment]
Yes i feel too but doesn't care, i use the javascript to make it responsive but need test real device not by resize the browser here is i have blog 
http://sidaurukfreddy.blogspot.com/2015/07/making-style-newspaper-using-html.html
